useScroll/useEvent.js

```diff
@@ -449,15 +489,7 @@ const useEventVoice = (props, instances) => {
 
 			/* if the focused element is out of the viewport, find another spottable element in the viewport */
 			if (spotItemBounds[last] <= viewportBounds[first] || spotItemBounds[first] >= viewportBounds[last]) {
-				const nodes = Spotlight.getSpottableDescendants(scrollContainerNode.dataset.spotlightId);
-				for (let i = 0; i < nodes.length; i++) {
-					const nodeBounds = nodes[i].getBoundingClientRect();
-
-					if (nodeBounds[first] >= viewportBounds[first] && nodeBounds[last] <= viewportBounds[last]) {
-						Spotlight.focus(nodes[i]);
-						break;
-					}
-				}
+				focusSpottableNode(scrollContainerNode.dataset.spotlightId, viewportBounds, false);
```

Comment:
Let's use `getDeepSpottableDescendants` instead of this as we discussed.

Reply:
That's a concise answer. Thank you